File: JavaScript/8js 1/billboard.js
/*
|--------------------------------------------------------------------------
| Name on billboard
|--------------------------------------------------------------------------
|
*/

function billboard(name, price = 30) {
   let res = 0;
   for (let i = 0; i < name.length; i++) res += price;
   return res;
}


function billboard(name, price = 30) {
   return name.length / (1 / price);
}


function billboard(name, price = 30) {
   return [...name].reduce((pre) => pre + price, 0);
}


function billboard(name, price = 30) {
   return name.repeat(price).length;
}


console.log(billboard("Jeong-Ho Aristotelis"), 600);
console.log(billboard("Abishai Charalampos"), 570);
console.log(billboard("Idwal Augustin"), 420);
console.log(billboard("Hadufuns John", 20), 260);
console.log(billboard("Zoroaster Donnchadh"), 570);
console.log(billboard("Claude Miljenko"), 450);
console.log(billboard("Werner Vígi", 15), 165);
console.log(billboard("Anani Fridumar"), 420);
console.log(billboard("Paolo Oli"), 270);
console.log(billboard("Hjalmar Liupold", 40), 600);
console.log(billboard("Simon Eadwulf"), 390);

File: Python/8py 1/billboard.py
"""
|--------------------------------------------------------------------------
| Name on billboard
|--------------------------------------------------------------------------
|
"""

def billboard(name, price=30):
   return sum([price for i in name])


def billboard(name, price=30):
   return len(name) / (1/price)


def billboard(name, price=30):
   res = 0
   for i in name: res += price
   return res


print(billboard("Jeong-Ho Aristotelis"), 600)
print(billboard("Abishai Charalampos"), 570)
print(billboard("Idwal Augustin"), 420)
print(billboard("Hadufuns John",20), 260)
print(billboard("Zoroaster Donnchadh"), 570)
print(billboard("Claude Miljenko"), 450)
print(billboard("Werner Vigi",15), 165)
print(billboard("Anani Fridumar"), 420)
print(billboard("Paolo Oli"), 270)
print(billboard("Hjalmar Liupold",40), 600)
print(billboard("Simon Eadwulf"), 390)